nixos/modules/services/networking/vsftpd.nix

```diff
@@ -163,6 +167,13 @@ in
         description = "Anonymous write umask.";
       };
 
+      extraConfig = mkOption {
+        type = types.string;
```

Comment:
`types.string` is deprecated. `types.str` is preferred for string values, but I think `types.lines` might be more appropriate here.